examples/data-sources/ybm_vpc/data-source.tf

```diff
@@ -0,0 +1,4 @@
+
+data "ybm_vpc" "example_vpc" {
+  name = "my-vpc"
```

Comment:
Can you please add an example for fetching by ID?

Reply:
Done @cdavid